app/server/src/app.ts

```diff
@@ -0,0 +1,69 @@
+import express from 'express';
+import winston from 'winston';
```

Comment:
I need to read the docs on this lib

Reply:
The server boilerplate is very WIP, just looking to get a base merged in to start working on it